examples/print-everything/src/main.rs

```diff
@@ -0,0 +1,66 @@
+// example
+
+use arctic::H10MeasurementType;
+
+struct Handler;
+
+#[arctic::async_trait]
+impl arctic::EventHandler for Handler {
+    async fn battery_update(&self, battery_level: u8) {
+        println!("Battery: {}", battery_level);
+    }
+
+    async fn heart_rate_update(&self, _ctx: &arctic::PolarSensor, heartrate: arctic::HeartRate) {
+        println!("Heart rate: {:?}", heartrate);
+    }
+
+    async fn measurement_update(&self, _ctx: &arctic::PolarSensor, data: arctic::PmdRead) {
+        println!("data: {:?}", data.data());
+    }
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let mut polar = arctic::PolarSensor::new("9F8F7521".to_string())
+        .await
+        .unwrap();
+
+    while !polar.is_connected().await {
+        match polar.connect().await {
+            Err(arctic::Error::NoBleAdaptor) => {
+                println!("No bluetooth adapter found");
+                return Ok(());
+            }
+            Err(why) => println!("Could not connect: {:?}", why),
```

Comment:
Ditto for looping connect. Sorry for stacking all these, just don't want to miss one.

Reply:
No problem